### PR TITLE
Update console script for hypershift cluster

### PR DESCRIPTION
**Pull request by yanpzhan**
@yapei @LiangquanLi930 pls help to review, thanks!

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
i am ok for hypershift part, @yanpzhan thanks for your support

**Comment by LiangquanLi930** [comment]
https://github.com/openshift/release/pull/33591 fix: kubedamin => kubeadmin

**Comment by LiangquanLi930** [comment]
@kuiwang02 could you help approve this PR, thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml [hunk 1]
```diff
@@ -87,6 +87,7 @@ tests:
       TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;HyperShiftGUEST&
       TEST_TIMEOUT: "30"
     test:
+    - ref: openshift-extended-web-tests
```
Comment (kuiwang02) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:90]: @yanpzhan please put it after openshift-extended-test ,like openshift-e2e-test-qe

**Pull request merged**